README.rst

```diff
@@ -482,20 +482,26 @@ Example that shows how to set custom registry
         pass
 
 
+    print("Before tests:", responses.mock.get_registry())
```

Comment:
Can we get these removed?

Reply:
we can, but why?

I think this is a good example to show how to check registry type